data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Agumon', '2020-02-3', 'Yes', 10.23, 0);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Gabumon', '2018-11-15', 'Yes', 8, 2);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Pikachu.', '2021-01-7', 'No', 15.04, 1);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Devimon', '2017-05-12', 'Yes', 11, 5);
```

Comment:
- Optional
Even though SQL accepts **yes** and **no** as boolean values, I would strongly recommend using **true** and **false** as per the community standards.